appveyor.yml

```diff
@@ -7,24 +7,37 @@ environment:
   VCINSTALLDIR: C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build
   VS150COMNTOOLS: C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build
   VSVARS32FILE: C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\vcvars32.bat
-  MSVC_VER: 14.12.25827
+  # MSVC_VER: 14.12.25827
   APPVEYOR_CACHE_ENTRY_ZIP_ARGS: "-t7z -m0=lzma -mx=9"
+  matrix:
+  - ARCHITECTURE: x64
+    MSVC_VER: 15
+    APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+
+matrix:
+  fast_finish: true
 
 shallow_clone: true
 
+clone_depth: 5                      # clone entire repository history if not defined
+
+
 build:
-  verbosity: detailed
+  verbosity: normal
+  # verbosity: detailed
 
 build_script:
   - ps: |
       choco install jdk9 --version 9.0.4.11 -params 'installdir=C:\\jdk9'
       choco install gradle --version 4.3.0
       refreshenv
-  - cmd: |
-      gradle build -PCOMPILE_WEBKIT=false --stacktrace -x :web:test --info --no-daemon
+      # detailed gradle build: add --info
+      gradle build -x :web:test -PCOMPILE_WEBKIT=false --no-daemon --stacktrace
+
+# Full test crash
+#test_script:
+#  - gradle test -x :web:test -PCOMPILE_WEBKIT=false --no-daemon --stacktrace
```

Comment:
Removing this line makes the test directories non-existent, so the test results uploading will fail, because no tests are being run.